src/adapter/aws-lambda/handler.ts

```diff
@@ -231,7 +236,11 @@ export abstract class EventProcessor<E extends LambdaEvent> {
     return new Request(url, requestInit)
   }
 
-  async createResult(event: E, res: Response): Promise<APIGatewayProxyResult> {
+  async createResult(
+    event: E,
+    res: Response,
+    options: { contentTypesAsBinary: string[] } = { contentTypesAsBinary: [] }
```

Comment:
Using the type definition in `HandleOptions` for `contentTypesAsBinary` is better.

```ts
options: Pick<HandleOptions, 'contentTypesAsBinary'> = { contentTypesAsBinary: [] }
```